infra/recipes/terraform/azure/modules/helm_release/main.tf

```diff
@@ -0,0 +1,55 @@
+resource "helm_release" "tables" {
+  chart = "tables"
+  name = "openhouse-tables-service"
+  repository = "../../../../k8s/helm"
```

Comment:
Maybe we can specify the root path prefix like `../../../../`? This is hard to read.

Reply:
I will add as a local variable